plugin/src/main/java/io/jenkins/plugins/casc/BaseConfigurator.java

```diff
@@ -52,7 +52,7 @@
 
     private static final Logger LOGGER = Logger.getLogger(BaseConfigurator.class.getName());
 
-    @NonNull
+    @Nonnull
```

Comment:
🤔 I still think this applies: https://blog.codefx.org/java/jsr-305-java-9/#Alternatives-To-JSR-305

Reply:
As far as I can tell the old ones are deprecated and point to these unless I’m missing something?

Reply:
NonNull should not be deprecated: https://github.com/spotbugs/spotbugs/issues/130

Reply:
K will look tomorrow at why we don’t have the spotbugs version

Reply:
It's coming from lib-version-number in jenkins-core by the looks of it, its fixed on master but un released

https://github.com/jenkins-infra/repository-permissions-updater/pull/1434

Will revert the changes here, and depending on timing bump the versions in this PR so no deprecation introduced here, or do in a follow up